### PR TITLE
Update README.md:

### PR DESCRIPTION
**Pull request by PaperL**
Correct typos with Grammarly;
Change some Markdown grammars;
Change some instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,33 @@
 
 # Summer-Ray-Tracer
 
-SJTU ACM Honor Class 2023 Ray Tracer Project With Rust !
+> SJTU ACM Honors Class 2023 Ray Tracer Project with Rust!
```
Comment (Alan-Liang) [inline README.md:5]: It is not really a quotation. Unnecessary formatting.

File: README.md [hunk 2]
```diff
@@ -2,31 +2,33 @@
...
 ### Introduction
 
-Briefly, you need to build a toy ray tracer in this project, with the Rust language.
+Briefly, you need to build a toy ray tracer in this project with the Rust language.
 
-### Where can you learn about ray tracing?
+### Where can you Learn about Ray Tracing?
 
-Out project is based on the [Ray Tracing In One Weekend](https://raytracing.github.io/) book series, containing three books, which is great resource for beginners to learn about ray tracing.
+Our project is based on the [***Ray Tracing In One Weekend***](https://raytracing.github.io/) book series, containing three books, which is a great resource for beginners to learn about ray tracing.
```
Comment (Alan-Liang) [inline README.md:15]: Unnecessary formatting. Avoid **bold** and *italic* text, especially ***bold and italic*** text.

File: README.md [hunk 3]
```diff
@@ -97,82 +99,81 @@ This is probably your first time to use Rust, so there are some resources for yo
...
+- Some [CG-related techniques](./doc/CG-bonus.md)
+- Some other interesting features. Please contact TAs if you have any ideas
 
-#### Task 5 (10 pts) : Final scene
+#### Task 5 (10 pts): Final scene
 
-- Complete a final scene, which should be more complex than the scenes in the books. If your work looks pretty or has aesthetic value, you will get extra points.
-- Or, your final scene can just be a logo of our project : `Ray Tracing`. Look at the current logo at the top of this page, you can make a better one, with your own ray tracer.
+- Complete a final scene, which should be more complex than the scenes in the books. If your work looks pretty or has aesthetic value, you will get extra points
+- Or, your final scene can just render our project name `Ray Tracing`. Look at the current banner at the top of this page, you can make a better one, with your own ray tracer
```
Comment (Alan-Liang) [inline README.md:157]: Meaning is changed; need further review. cc @hnyls2002

File: README.md [hunk 4]
```diff
@@ -182,16 +183,16 @@ Just do some preparation work, including:
...
     - **Day 7 - 9:** Task 2 & 5
     - **Day 10:** Pre & Task 6
 
-### Others
+### More Tips
 
-- You should have a document recording all the bonus you have done. For each bonus, you should show the code or the effect of your work.
-- There is no restriction on unsafe code, if you think it is necessary, you can use it, and explain why.
-- There will be 2 students to do presentations for ACM Class and SEIEE CS. They will get extra points based on their presentations, no more than 10 points.
+- You should have a document recording all the bonus you have done. For each bonus, you should show the code or a benchmark of your work
```
Comment (Alan-Liang) [inline README.md:196]: The list is comprised of full sentences. Full sentences should be ended by full stops.

Comment (hnyls2002) [reply README.md:196]: Ok

File: README.md (hunk 3 above)
Comment (hnyls2002) [reply README.md:157]: Ok, either a logo or a banner would be fine.
